rllib/utils/test_utils.py

```diff
@@ -979,6 +979,50 @@ def check_train_results(train_results: ResultDict):
     return train_results
 
 
+def should_stop(
```

Comment:
nit: Add TODO to also use this new util in our example script util function (if not using tune, `--no-tune`).

Reply:
Will do. Add the method then in another PR later